content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
+draft = false
+image = "/images/2021/09/laptop-plant.jpg"
+slug = "interview-process"
+tags = ["Careers", "Software Craftsmanship", "TDD"]
+title = "How we hire Software Craftspeople"
++++
+
+Incubyte is a combined effort of its team. Working here entails bringing your knowledge, curiosity, and authenticity to the table. Our recruiting process plays a vital role here, and it is carefully designed to identify the right person whose technical prowess, culture, and mindset, all align with that of the Company.
+
+We care about our core values ([Why Incubyte](https://blog.incubyte.co/blog/why-incubyte/)) and are constantly working to create a diverse and nurturing workplace while enabling a culture of constant learning and feedback. A place where each team member can thrive, and be the best version of themselves while making steady progress towards personal and professional mastery. Read on to know about our hiring process.
+
+A typical interview process at Incubyte is divided into 5 stages:
+
+1. Application
+2. Pre-required learning
+3. Call with the People Success Team
+4. Technical Discussion Round
+5. Software Craft Round
+
+## Pre-required Learning
+After you send in your application, and your resume has been screened by the People Success Team, you will receive an email that lists a few concepts and short videos. This step exists to align expectations through the interview process. This pre-requisite learning email will equip you with resources that set you up for success and.. we give you unlimited time to go through it!
+
+Here are some of the things we recommend: 
+- [Test-Driven Development](https://www.youtube.com/watch?v=qkblc5WRn-U)
+- [Software Craftsmanship](https://www.youtube.com/watch?v=c07uQGBZl0A)
+- [Clean Code](https://www.youtube.com/watch?v=zV079g7Irks&feature=emb_imp_woyt)
+
+This step also shines light on other skills needed to clear the interviews. For example, knowing common Git operations, hands-on knowledge of your IDE (with shortcuts), and an understanding of a unit testing framework to name a few!
+
+## Call with the People Success Team
+In this step, we aim to align on the role you are looking for and what we have to offer and see if there is synergy there. Being remote-first also means that it is important that we align not just on tech; but also on culture, attitude, and core values! Our People Success Team are here to listen to your goals and help answer any questions you may have about the Company. Ask away!
+
+## Technical Discussion with our existing team members
+A 30-minute discussion around your technical background and journey so far. You can expect the conversation to revolve around languages, frameworks, and databases you have worked with; your prior experiences, projects, etc. This is for us to get introduced to the depth and breadth of your technical prowess. Here it is important that you have a good Wi-Fi connection since we'd want to have your video on during the discussion.
```

Comment:
Does this work? A 30-minute discussion focused on your technical background and journey so far. You can expect the conversation to revolve around languages, frameworks, and databases you have worked with; your prior experiences, projects, etc. This is for us to get introduced to the depth and breadth of your technical expertise. **_It is important that you have a good Wi-Fi connection since we'd want to have your video on during both, this technical discussion and the next hands-on pairing session._**